app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -5,17 +5,26 @@ name: iPXE intermediate script
 snippet: false
 -%>
 #!ipxe
-# Intermediate iPXE script to report MAC address to Foreman
 
-<% (0..32).each do |i| -%>
-:net<%= i %>
-isset ${net<%= i -%>/mac} || goto no_nic
-dhcp net<%= i -%> || goto net<%= i + 1 %>
-chain <%= foreman_url('iPXE') %>?mac=${net<%= i -%>/mac} || goto net<%= i + 1 %>
+# Skips booting from network and continues booting from next device
+exit
```

Comment:
I belive this change does not make sense. If I‘m not mistaken the change in the core repo is the one we want to have.

Reply:
Could you please open a PR to community templates with it? Look like in core it was added by https://github.com/theforeman/foreman/pull/6061 while in community templates a different version was added in https://github.com/theforeman/community-templates/pull/512.

Reply:
https://github.com/theforeman/community-templates/pull/578

Reply:
Thanks, updated. Any idea about the snapshots failing on `pxegrub2_chainload`?

Reply:
Taking a look, in the meantime I created a PR for the typo: https://github.com/theforeman/community-templates/pull/579

Reply:
It tries to find the snipped with the source classs `Foreman::Renderer::Source::Database`, however it should use `SnapshotSource` instead.

You can debug that with:
```ruby
diff --git a/lib/foreman/renderer/scope/macros/snippet_rendering.rb b/lib/foreman/renderer/scope/macros/snippet_rendering.rb
index d5db6f787..adae111d0 100644
--- a/lib/foreman/renderer/scope/macros/snippet_rendering.rb
+++ b/lib/foreman/renderer/scope/macros/snippet_rendering.rb
@@ -15,6 +15,7 @@ module Foreman
           def snippet(name, options = {}, variables: {})
             template = source.find_snippet(name)
             unless template
+              binding.pry if name == 'pxegrub2_discovery'
               raise "The specified snippet '#{name}' does not exist, or is not a snippet." unless options[:silent]
               return
             end
```

I haven't figured out yet, where the wrong source comes from.

Reply:
I am looking into another problem - discovery menus are nested by accident.

Reply:
The `Foreman::Renderer::Source::Database` source comes from this:
https://github.com/theforeman/foreman/blob/3f4943103cf73274c827bfe61b7b14cc4ff0c2fb/lib/foreman/renderer.rb#L31-L33

However I ran out of time for now and wasn't able to provide a fix for this.

Reply:
One observation: pxegrub2_discovery is now included from another snippet named pxegrub2_chainload. Can this be the issue (snippet-snippet context wrong)?